streams/src/main/java/org/apache/kafka/streams/processor/internals/assignment/HighAvailabilityTaskAssignor.java

```diff
@@ -34,7 +33,6 @@
 import java.util.UUID;
 import java.util.stream.Collectors;
 
-import static org.apache.kafka.streams.processor.internals.assignment.AssignmentUtils.taskIsCaughtUpOnClientOrNoCaughtUpClientsExist;
```

Comment:
If we no longer need this in the HATA then I believe it's only needed in `TaskMovement`. Should we just move it there (and the associated tests to `TaskMovementTest`)?

Reply:
I removed `AssignmentUtils` and did NOT move the tests to `TaskMovementTest`. Since now `taskIsCaughtUpOnClientOrNoCaughtUpClientsExist` it is a private method, I guess it is OK.

Reply:
Sounds good. I would imagine we have a test that fails if that does not work as intended?